plugin.xml

```diff
@@ -29,9 +29,10 @@
     <engine name="cordova-ios" version="3.9.2" />
   </engines>
 
-  <dependency id="cordova-plugin-whitelist" />
```

Comment:
No - they're all still there. It's just one line lower because of the comment line I added.